addons/hr_attendance/static/src/hr_attendance.scss

```diff
@@ -58,3 +58,9 @@
         image: var(--homeMenu-bg-image, url("/hr_attendance/static/img/background-light.svg"));
     }
 }
+
+.o_pin_code_btn {
```

Comment:
since there is already a specific class `o_hr_attendance_PINbox_button` only used for this element, can't we just use that? there is also already CSS statement for that class in this file:

https://github.com/odoo/odoo/blob/ad997b2a629a002a2c44123cac2b4ff47d0a7f1a/addons/hr_attendance/static/src/hr_attendance.scss#L33